docs/video-miners/getting-started/activation.md

```diff
@@ -1,185 +1,178 @@
-    find the 100th orchestrator with the most stake. This is the amount of stake
-    required for you to activate. Note that more stake will help you receive
-    more work
-
-## Activate
-
-The active orchestrator set consists of the top 100 orchestrators with the most
-LPT stake on the network.
-
-Once you have your orchestrator running, you can activate using `livepeer_cli`.
-
-1. Run `livepeer_cli`
-
-2. Enter the number corresponding to the
-   `Invoke multi-step "become an orchestrator"` option
-
-3. Set the percentage of LPT rewards that you will keep (the rest will be shared
-   with your delegators):
-
-   ```bash
-   Enter block reward cut percentage (current=0 default=10) - >
-   ```
-
-4. Set the percentage of ETH fees that you will keep (the rest will be shared
-   with your delegators):
-
-   ```bash
-   Enter fee cut percentage (current=100 default=95) - >
-   ```
-
-5. Set the number of pixels in a single unit of work you will charge a price
-   for:
-
-   ```bash
-   Enter amount of pixels that make up a single unit (default: 1 pixel) >
-   ```
-
-   You can use the default (1) and change this later if needed.
-
-6. Set the price (in wei) that you will charge per unit of work:
-
-   ```bash
-   Enter the price for 1 pixels in Wei (required) >
-   ```
-
-   This can be the same as or different from the value used with the
-   `-pricePerUnit` flag when starting your orchestrator.
-
-7. Set the publicly accessible service address that your orchestrator will
-   receive requests as:
-
-   ```bash
-   Enter the public host:port of node (default: 1.1.1.1:8935)>
-   ```
-
-   This should be the same as the value used with the `-serviceAddr` flag when
-   starting your orchestrator.
-
-8. Set the amount of LPTU (1 LPT = 1e18 LPTU) that you want to stake:
-
-   ```bash
-   You must bond to yourself in order to become a orchestrator
-   Enter bond amount - >
-   ```
-
-   **Important note:** This amount is denominated in LPTU, so if you want to bond 5 LPT,
-   you would enter 5000000000000000000.
-
-   If the active orchestrator set is full (i.e. at 100), the minimum stake you
-   need to stake to activate is the lowest total stake of an orchestrator in the
-   active set. You can find this information in the
-   [explorer](https://explorer.livepeer.org/).
-
-9. Wait for transactions to confirm. You should see the logs of your
-   orchestrator indicating transactions being submitted and confirmed on-chain.
-
-   After the transactions have confirmed, your orchestrator will join the active
-   set in the following round.
-
+---
+title: Activation
+sidebar_position: 4
+---
+
+This guide provides steps to activate on the Livepeer network so that you can receive video streams from broadcasters in order to provide transcoding services. You will set several parameters, including but not limited to the onchain price and address broadcasters use to find an orchestrator/node.

```

Comment:
this sentence reads *slightly* weird to me but I can't figure out why.

> This guide provides steps to activate on the Livepeer network so that you can receive video streams from broadcasters in order to provide transcoding services.

Reply:
i will work on it